strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/GroupRepositoryProvider.java

```diff
@@ -99,7 +101,17 @@ public ArtifactInputStream getInputStream(String storageId,
                 !repositoryRejects(r.getId(), artifactPath, denyRules) &&
                 !repositoryRejects(r.getId(), artifactPath, wildcardDenyRules))
             {
-                final ArtifactInputStream is = resolveArtifact(sId, r.getId(), artifactPath);
+                ArtifactInputStream is;
+                try
+                {
+                    is = resolveArtifact(sId, r.getId(), artifactPath);
+                }
+                catch (FileNotFoundException e)
+                {
+                    logger.debug(String.format("Artifact not found in repository, continue group artifact search: repo-[%s]; path-[%s].",
```

Comment:
We don't need this extra output. If it's not found, that's absolutely okay. If this is reported by Sonar, please, ignore the warning.